src/main.c

```diff
@@ -460,6 +465,160 @@ void WINAPI EventCallback(PEVENT_RECORD ev)
     }
 }
 
+static void BluetoothHciEventCallback(PEVENT_RECORD ev)
```

Comment:
Most of this function is copy-pasted from the EventCallback function. Wouldn't this be cleaner with a single EventCallback?